units/systemd-userdb-load-credentials.service

```diff
@@ -0,0 +1,31 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Load JSON user/group records from Credentials
+Documentation=man:systemd-userdb(8)
+Documentation=man:systemd.system-credentials(7)
+
+DefaultDependencies=no
+Before=systemd-user-sessions.service
+After=local-fs.target systemd-journald.socket
+Conflicts=shutdown.target
+Before=shutdown.target
+
+ConditionPathExists=!/etc/initrd-release
```

Comment:
we should add globbing to ConditionCredential= and then match against userdb.* here

Reply:
I assume you don't mean this should be done in this PR